test/module/shared_model/validators/commands_validator_test.cpp

```diff
@@ -167,10 +191,13 @@ TEST(CommandsValidatorTest, StatelessValidTest) {
  * @then answer has errors and number of errors in answer is the same as the
  * number of commands in tx
  */
-TEST(CommandsValidatorTest, StatelessInvalidTest) {
+TEST_F(CommandsValidatorTest, StatelessInvalidTest) {
   iroha::protocol::Transaction tx = generateEmptyTransaction();
   auto payload = tx.mutable_payload();
 
+  iroha::ts64_t invalid_time = 10000000000ull;
+  payload->set_created_time(invalid_time);
```

Comment:
Why does it do in that pr?

Reply:
Because I updated generate empty transaction method and now it returns valid tx with valid created time. WHile in this test we want to check if validator will throw error when invalid created time is set